teach/templates/clubs/email/create_club_staff/body.txt

```diff
@@ -0,0 +1,14 @@
+{{ username }} would like to add their Club to the map.
```

Comment:
We probably want to change this to `{{ full applicant name}} ({{ username }}) would like ...` then, since we're asking for people's full names.